jetty-http2/http2-client/src/test/java/org/eclipse/jetty/http2/client/RawHTTP2ProxyTest.java

```diff
@@ -415,12 +413,9 @@ protected Action process() throws Throwable
                     @Override
                     public void succeeded(Stream result)
                     {
-                        synchronized (lock)
-                        {
-                            if (LOGGER.isDebugEnabled())
-                                LOGGER.debug("CPS created {}", result);
-                            streams.put(clientToProxyStream, result);
-                        }
+                        if (LOGGER.isDebugEnabled())
+                            LOGGER.debug("CPS created {}", result);
```

Comment:
this debug is now outside of the lock

Reply:
Yes, it only references local variables and does not need to be inside the lock.